test/pjrt/test_experimental_pjrt_tpu.py

```diff
@@ -250,6 +252,33 @@ def test_device_attributes(self):
       self.assertIsInstance(device['coords'], list)
       self.assertIsInstance(device['core_on_chip'], int)
 
+  @staticmethod
+  def _execute_time_metric():
+    # Initialize the client before starting the timer.
+    xm.xla_device()
+
+    begin = time.perf_counter_ns()
+    value = (
+        torch.randn(10000, 10000, device=xm.xla_device()) *
+        torch.randn(10000, 10000, device=xm.xla_device()))
+    value_mean = value.mean()
+    xm.mark_step()
+    cpu_value = value_mean.cpu()
+    wall_time_ns = time.perf_counter_ns() - begin
+    _, execute_time_ns, _ = met.metric_data('ExecuteTime')
+
+    return execute_time_ns
+
+  def test_execute_time_metric(self):
+    results = pjrt._run_multiprocess(self._execute_time_metric)
```

Comment:
is this test the one added by Goran?

Reply:
Goran added a similar test for CPU here:

https://github.com/pytorch/xla/blob/9c01c3ca14f7b34f84421c6db1e102386c26cf3f/test/test_metrics.py#L166-L186

Making that test work for TPU is tricky because the overhead is way higher on TPU, so the execute time is a small fraction of the overall wall time. I did check that this new test does fail if I revert our fix to the execute time metric.